java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -26,6 +26,7 @@
 import static redis_request.RedisRequestOuterClass.RequestType.LPop;
 import static redis_request.RedisRequestOuterClass.RequestType.LPush;
 import static redis_request.RedisRequestOuterClass.RequestType.LRange;
+import static redis_request.RedisRequestOuterClass.RequestType.LRem;
```

Comment:
wouldn't it make more sense to import `redis_request.RedisRequestOuterClass.RequestType` once, instead of importing each one of its (eventual) 300 values?

Reply:
We could do that in a later PR.